hera_qm/tests/test_xrfi.py

```diff
@@ -120,7 +131,7 @@ def test_medminfilt(self):
         # test cases where filters are larger than data dimensions
         Kt = self.size + 1
         Kf = self.size + 1
-        d_filt = uvtest.checkWarnings(xrfi.medminfilt, [data, Kt, Kf], nwarnings=2,
+        d_filt = uvtest.checkWarnings(xrfi.medminfilt, [data], {'Kt': Kt, 'Kf': Kf}, nwarnings=2,
```

Comment:
There were a bunch of places we were using positional arguments for things that should be keywords.